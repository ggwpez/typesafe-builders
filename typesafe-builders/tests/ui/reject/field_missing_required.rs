use typesafe_builders::prelude::*;

#[derive(Builder)]
struct Struct {
	x: u8,
}

fn main() {
	Struct::builder().build();
}
