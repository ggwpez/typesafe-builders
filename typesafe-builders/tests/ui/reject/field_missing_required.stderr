error[E0599]: no method named `build` found for struct `GenericStructBuilder<false>` in the current scope
 --> tests/ui/reject/field_missing_required.rs:9:20
  |
3 | #[derive(Builder)]
  |          ------- method `build` not found for this struct
...
9 |     Struct::builder().build();
  |                       ^^^^^ method not found in `GenericStructBuilder<false>`
  |
  = note: the method was found for
          - `GenericStructBuilder<true>`
